icu4c/source/common/cmemory.h

```diff
@@ -101,7 +101,7 @@ uprv_calloc(size_t num, size_t size) U_MALLOC_ATTR U_ALLOC_SIZE_ATTR2(1,2);
   *    Clears any user heap functions from u_setMemoryFunctions()
   *    Does NOT deallocate any remaining allocated memory.
   */
-U_CFUNC UBool 
```

Comment:
The ICU4C code base has thousands of lines with trailing whitespace, making one random clean-up to one of them here on a line unrelated to the rest of the changes in the commit is just noise. Please revert this line.

Reply:
Thanks for answering the implied question: automatically trimming whitespace of edited files is thus not a useful IDE setting. I'll change it!

Reply:
Done.